content/en/blog/getting-started/index.md

```diff
@@ -0,0 +1,252 @@
+
+We assume that the holder operates a web wallet and has taken it into use (**8-9**).
+
+```mermaid
+sequenceDiagram
+  autonumber
+    participant CLI
+    participant Agency
+    participant Web Wallet
+
+    CLI->>Agency: Register
+    CLI->>Agency: Login
+    Agency-->>CLI: JWT token
+    CLI->>Agency: Create schema
+    Agency-->>CLI: Schema id
+    CLI->>Agency: Create cred def
+    Agency-->>CLI: Cred def id
+    Note left of Web Wallet: Holder takes the web wallet in use
+    Web Wallet->>Agency: Register
+    Web Wallet->>Agency: Login
+```
+
+### CLI Script Issuing Credential
+
+The next task is to create a pairwise connection between the agent operated by the CLI user
+and the web wallet user. The pairwise connection is an encrypted pipe between the two
+agents that they can use to exchange data securely. The CLI script creates an invitation (**1-2**)
+and prints it out (**3**) as a QR code that the web wallet user can read (**5**).
+
+Once the new connection id is known, the CLI script starts a chatbot (**4**) for the new connection.
+The bot logic follows the rules for changing the bot states in the YAML configuration file. Therefore,
+the bot handles the rest of the issuing process (**6-7**).
+
+Once the issuer bot notices that credential issuing succeeded, it stops the bot (**10-11**), and
+the sample script moves on to verifying the same credential.
+
+```mermaid
+sequenceDiagram
+  autonumber
+    participant CLI
+    participant Issue Bot
+    participant Agency
+    participant Web Wallet
+
+    CLI->>Agency: Create invitation
+    Agency-->>CLI: Invitation URL
+    CLI-->>Web Wallet: <<show QR code>
+    CLI->>Issue Bot: Start
+    Web Wallet->>Agency: Read QR code
+    Agency-->>Issue Bot: Connection ready!
+    Issue Bot->>Agency: Issue credential
+    Agency-->>Web Wallet: Accept offer?
+    Web Wallet->>Agency: Accept
+    Agency-->>Issue Bot: Issue ready!
+    Issue Bot->>Issue Bot: Terminate
+```
+
+### CLI Script Verifying Credential
+
+Steps **1-6** proceed similarly to the issuing: first, the agents form a new pairwise connection.
+However, the process continues with a proof request sent by the verifier bot (**7**). The proof request
+contains the attributes the bot wishes the holder to reveal. The web wallet user sees
+the requested data once they receive the message (**8**), and they can either accept or reject the request.
+
+Once the proof is accepted (**9**), the agency verifies it cryptographically.
+If the verification succeeds, the agency notifies the verifier bot with the proof values (**10**).
+It can reject the proof if the values are not acceptable by the business logic.
+The sample bot accepts all attribute values, so the verifying process is continued
+without extra validation (**11**).
+The bot exits when the proof is completed (**12-13**).
+
+And that’s all core features of verified data flow executed utilizing the CLI tool!
```

Comment:
Just a comment to explain my next comment: here, you start to feel that we have reached the end. I mean as a reader.

Reply:
^ I mean my comment about splitting. Next level 2 heading.